frontend/www/js/omegaup/components/common/NavbarItems.vue

```diff
@@ -171,6 +171,11 @@
             href="https://www.youtube.com/playlist?list=PLdSCJwXErQ8FhVwmlySvab3XtEVdE8QH4"
             >{{ T.navTutorials }}</a
           >
+          <a
+            class="dropdown-item text-wrap"
+            href="https://discord.gg/Y4JQ5HQd"
+            >{{ T.discordLink }}</a
```

Comment:
the surrounding entires (`navTutorials` and `navBlog`) start with the `nav` prefix. for consistency and to aid in finding this entry, let's rename it

```suggestion
            >{{ T.navDiscord }}</a
```

Reply:
@lhchavez I have to changed this only at NavbarItems.vue file or other files too? I am asking this because I have added   in 12-13 files.

Reply:
You need to replace it in every place you are using it

Reply:
Btw, you don't need to ping lhchavez, because he receives an email with all your comments

Reply:
> You need to replace it in every place you are using it

ohkk

Reply:
> Btw, you don't need to ping lhchavez, because he receives an email with all your comments

ohh sorry for that,  I will remember this and in future it will not happen again.

Reply:
I have made changes which you have asked. Now you can check it.

Reply:
This look good to me. I have only a comment:

Your change has a conflict in the file `NavbarItems.vue`. The way to fix it is fetching and then merging with main branch:
```
git fetch upstream main:main
git merge upstream/main
```
Once you have merged your branch, you need to make some changes manually in the file `NavbarItems.vue`, specifically in https://github.com/omegaup/omegaup/pull/6506/conflicts

Finally you have to commit those last changes and push them.

Reply:
I have run the suggested commands and after which I manually make changes in the file NavbarItems.vue . Please have a look